package.json

```diff
@@ -52,9 +53,9 @@
     "test:generate-routes": "turbo run test:generate-routes --filter=!pie-monorepo --token=${TURBO_TOKEN}",
     "test:browsers-setup": "turbo run test:browsers-setup --filter=!pie-monorepo --token=${TURBO_TOKEN}",
     "test:browsers": "cross-env-shell turbo run test:browsers --filter=!pie-monorepo --token=${TURBO_TOKEN}",
-    "test:browsers:ci": "cross-env-shell turbo run test:browsers:ci --filter=!pie-monorepo --token=${TURBO_TOKEN}",
+    "test:browsers:ci": "cross-env-shell turbo run test:browsers:ci --filter=!pie-monorepo --token=${TURBO_TOKEN} --env-mode=loose",
```

Comment:
By default, Turborepo now only provides env variables to task that are defined in `env` - https://turbo.build/repo/docs/reference/configuration#env


This is because env vars _should_ be considered as part of the cache hash. We should look to eventually update tasks to define the env vars they're dependant on, and remove this flag.